docs/code_standards.rst

```diff
@@ -84,13 +114,21 @@ Importing
 General
 
 * Avoid global variables.
-* Use :code:`.join()` for string concatenation.
+* Use :code:`.join()` for non-trivial string concatenation.
+* Trivial string concatenation can be done either via f-strings or `+`, though f-strings are preferred.
```

Comment:
I think `+` concatenation way is always consider bad way of doing it. Can we altogether disagree with `+` concatenation.

fstings solved it much better.

Reply:
I agree with you here @jyejare but was willing to accept it, as @JacobCallahan had some justification for performance.

Reply:
@mshriver Not exactly sure how much time it would really save (probably in milliseconds), but f-string has an upper-hand to plus concatenation when it comes to readability/ease.

Reply:
@jyejare I'll leave the guidance here as-is, since trivial string concatenation with `+` is both easier to write/read and more performant that f-strings.